### PR TITLE
Update readme to include new SPI subsets + SPI wishlist

### PR DESCRIPTION
**Pull request by anniegbryant**
This PR introduces a new SPI subset ('fabfour'), which includes four SPIs:

Pearson correlation
Spearman correlation
Directed information with a Gaussian density estimator
Power envelope correlation
As we now have three possible SPI subsets (fast, sonnet, fabfour), this PR also reconfigures the specification to be one argument to the Calculator object (subset) with the options of "all" (default), "fast", "sonnet", or "fabfour". The documentation has been updated to reflect this change.

The readme is also updated accordingly to reflect these subsets, as well as by starting an SPI wishlist.

### CODE REVIEW

**Comment by anniegbryant** [comment]
@benfulcher ready for your review now, this amends the subset syntax and directs users to the projects tab to suggest new SPIs

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Feedback is much appreciated through email, [issues](https://github.com/olivercl
 
 If you use this code, please cite the following preprint:
 
-Oliver M. Cliff, Joseph T. Lizier, Naotsugu Tsuchiya, Ben D. Fulcher, "Unifying Pairwise Interactions in Complex Dynamics," _arXiv_ preprint, [arXiv:2201.11941](https://arxiv.org/abs/2201.11941) (2022).
+Oliver M. Cliff, Annie G. Bryant, Joseph T. Lizier, Naotsugu Tsuchiya, Ben D. Fulcher, "Unifying Pairwise Interactions in Complex Dynamics," _arXiv_ preprint, [arXiv:2201.11941v2](https://arxiv.org/abs/2201.11941v2) (2023).
```
Comment (benfulcher) [inline README.md]: If you leave off the 'v2' from the URL then it takes to the most recent version. This is preferable

File: README.md [hunk 2]
```diff
@@ -40,3 +40,7 @@ If your dataset is large (containing many processes and/or observations), you ca
 ### _hcga_
 
 [_hcga_](https://github.com/barahona-research-group/hcga), a *highly comparative graph analysis* toolkit, computes several thousands of graph features directly from any given network.
+
+## SPI Wishlist
+
+As _pyspi_ is under active development, we are always open to suggestions for new SPIs to be added via the [projects tab](https://github.com/DynamicsAndNeuralSystems/pyspi/projects) in this repo.
```
Comment (benfulcher) [inline README.md:46]: Ooh cool idea—is it functional?

File: pyspi/calculator.py [hunk 3]
```diff
@@ -30,28 +30,30 @@ class Calculator:
...
+            elif subset == "sonnet":
                 configfile = (
                     os.path.dirname(os.path.abspath(__file__)) + "/sonnet_config.yaml"
                 )
+            elif subset == "fabfour":
+                configfile = (
+                    os.path.dirname(os.path.abspath(__file__)) + "/fabfour_config.yaml"
+                )
             else:
                 configfile = os.path.dirname(os.path.abspath(__file__)) + "/config.yaml"
```
Comment (benfulcher) [inline pyspi/calculator.py:64]: Imagine a user mis-spelling say 'fabfive'. Can you catch this instance with an appropriate message? E.g., "there is no feature subset named 'fabfive', …"

**Review by benfulcher — CHANGES_REQUESTED** [review]
A few minor comments—can be addressed pretty trivially…

File: README.md (hunk 2 above)
Comment (anniegbryant) [reply README.md:46]: Yep (kinda)! Check it out -- https://github.com/orgs/DynamicsAndNeuralSystems/projects/2

File: pyspi/calculator.py (hunk 3 above)
Comment (anniegbryant) [reply pyspi/calculator.py:64]: Added

Comment (benfulcher) [reply pyspi/calculator.py:64]: I don't mean fabfive explicitly, but any key that isn't in the current set of valid subset names

Comment (benfulcher) [reply pyspi/calculator.py:64]: This is the last pending change

**Review by benfulcher — CHANGES_REQUESTED** [review]
Make catch for invalid subset string more general

**Pull request merged**